OpenOversight/app/templates/browse.html

```diff
@@ -14,14 +14,20 @@ <h1><small>Browse a Department</small></h1>
         <h2>{{ department.name }}
           <p>
           <a class="btn btn-lg btn-primary" href="{{ url_for('main.list_officer', department_id=department.id) }}">
-             Officers
+             Browse Officers
           </a>
           {% if department.incidents %}
             <a class="btn btn-lg btn-primary" href="{{ url_for('main.incident_api', department_id=department.id) }}">
-               Incidents
+               Browse Incidents
             </a>
-        </p>
-        {% endif %}
+            <a class="btn btn-lg btn-primary" href="{{ url_for('main.download_incidents_csv', department_id=department.id) }}">
+               Download Incident Data
+            </a>
+          {% endif %}
+          <a class="btn btn-lg btn-primary" href="{{ url_for('main.download_dept_csv', department_id=department.id) }}">
+               Download Officer Data
+          </a>
+          </p>
```

Comment:
Not necessary for merge on this PR, but just a thought: we should have the officers and incidents links be primary, and the raw data downloads be somewhere less central. One good place on the site might be a link on the footer to a new route that has the CSV download links for each department:

<img width="739" alt="screen shot 2018-08-18 at 6 52 23 pm" src="https://user-images.githubusercontent.com/7832803/44304690-dfda4880-a317-11e8-90e8-6284a9ca53ac.png">

If you feel like implementing this (or if you have another better idea!) feel free, else we can defer to a followup.